pkg/component/component.go

```diff
@@ -7,11 +7,15 @@ import (
 	buildv1 "github.com/openshift/api/build/v1"
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+
+	"strings"
```

Comment:
please group this with other standard library imports

Reply:
fixed